pkg/cmd/grafana-cli/services/api_client_test.go

```diff
@@ -12,8 +12,9 @@ import (
 
 func TestHandleResponse(t *testing.T) {
 	t.Run("Returns body if status == 200", func(t *testing.T) {
-		body, err := handleResponse(makeResponse(200, "test"))
+		bodyReader, err := handleResponse(makeResponse(200, "test"))
 		assert.Nil(t, err)
+		body, _ := ioutil.ReadAll(bodyReader)
```

Comment:
Can you also please verify that `ioutil.ReadAll` doesn't return an error?

Reply:
Yes. I've also replaced `assert.Nil` by `assert.NoError`, which has a more relevant message.